nat-lab/tests/conftest.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+import asyncio
+import signal
```

Comment:
unused `signal` and `logging` imports, run `autoflake`, will remove unused imports automagically :smile: